src/toil/test/wdl/builtinTest.py

```diff
@@ -234,6 +243,14 @@ def check_function(self, function_name, cases, expected_result):
                 self.assertEqual(result, expected_result)
                 shutil.rmtree(output_dir)
 
+    def test_sub(self):
+        # this workflow swaps the extension of a TSV file to CSV, with String and File inputs.
+        self.check_function('sub', cases=['as_input'], expected_result='src/toil/test/wdl/test.csv')
+
+        # NOTE: the result differs from Cromwell since we copy the file to the file store without
+        # preserving the path. Cromwell would return 'src/toil/test/wdl/test.csv' instead.
+        self.check_function('sub', cases=['as_input_with_file'], expected_result='test.csv')
```

Comment:
I don't think we can produce the same behavior with our current implementation?

Reply:
Oh, interesting, you're right.  I don't think we need to address this immediately, and the comment should be enough for now, especially since `sub()` is still doing exactly what it should.